apps/prairielearn/src/ee/pages/administratorInstitutionSso/administratorInstitutionSso.html.ts

```diff
@@ -78,7 +78,7 @@ export const AdministratorInstitutionSso = ({
               None
             </option>
             ${supportedAuthenticationProviders.map((provider) => {
-              if (provider.name === 'LTI') return '';
+              if (provider.name === 'LTI' || provider.name === 'LTI 1.3') return '';
```

Comment:
Drive-by fix to an issue I noticed while testing. LTI sessions must always start from the LTI provider (we can't show a "sign in with UIUC Canvas" link), so they can't be a default option.

Reply:
nit: should we be reusing `isLtiProvider` here?

Reply:
I don't see a compelling reason to do so. That was introduced (locally) on the other page only because that check was repeated so many times.